Create ui-c8y-1023-48-3-stacked-drawers-escape-button-handling-graft-release-cd-11346.md (#4349)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-48-3-stacked-drawers-escape-button-handling-graft-release-cd-11346.md

---
date: ""
title: Stacked drawers now close only the top drawer when pressing escape
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65365
version: 1023.48.3
---

> When multiple drawers were displayed in a stacked manner, pressing the escape key would close all open drawers at once, which could be disruptive to the user workflow. Now, pressing escape will close only the topmost drawer, allowing users to navigate back through multiple drawer levels one at a time. Additionally, you can now configure individual drawers to prevent them from closing when the escape key is pressed, giving you more control over the user experience in your applications.

Reviewer: ```suggestion
When multiple drawers were displayed in a stacked manner, pressing the escape key closed all open drawers at once, which could disrupt the user's workflow. Now, pressing escape will close only the topmost drawer, allowing users to navigate back through multiple drawer levels one at a time. Additionally, you can now configure individual drawers to prevent them from closing when the escape key is pressed, giving you more control over the user experience in your applications.
```
